inc/saihostintf.h

```diff
@@ -148,16 +148,26 @@ typedef sai_status_t(*sai_get_hostif_trap_group_attribute_fn)(
         _Inout_ sai_attribute_t *attr_list);
 
 /**
+* @brief Host interface user defined trap type table range
+*/
+#define SAI_HOSTIF_TRAP_TYPE_TABLE_RANGE 0x1
```

Comment:
sai_hostif_trap_type_t is being used as the key and needs to be unique for each trap entry created. Having this SAI_HOSTIF_TRAP_TYPE_TABLE_RANGE value set to 1 limits the no.of traps to 2 for a particular type. How do we need to handle in case of table size more than 2 i.e 255 or 1024 entries ?